### PR TITLE
Fix additional clippy warnings

### PR DESCRIPTION
**Pull request by bjfish**

### CODE REVIEW

File: src/webassembly/utils.rs [hunk 1]
```diff
@@ -48,7 +48,7 @@ instance.data_pointers.globals \t- {:X} | offset - {:?}
         memories_pointer_address_0,
         0,
         memories_pointer_address_0_data,
-        memories_pointer_address_0_data - memories_pointer_address_0_data,
+        memories_pointer_address_0_data - instance_address,
```
Comment (bjfish) [inline src/webassembly/utils.rs]: I assume this is `instance_address` to match the rest.

Comment (syrusakbary) [reply src/webassembly/utils.rs]: Good call. Haven't realized of that typo

**Comment by syrusakbary** [comment]
@bjfish I tried to merge master manually but it resulted in cargo not passing.

**Comment by syrusakbary** [comment]
Great! Merging

**Pull request merged**